From e170a76bc9f84ef7a92e6ac64f286afecd448afd Mon Sep 17 00:00:00 2001
From: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>
Date: Sat, 24 May 2025 20:30:56 +0200
Subject: [PATCH] metainfo: Wrap old release notes in paragraph

This fixes the following warnings in `appstreamcli validate`:

W: org.gnome.Calls:239: description-spurious-text
   The description element contains raw text that is not in any paragraph or other permitted tag.
   This is not allowed and the additional text may be ignored by parsers or raise errors.

W: org.gnome.Calls:239: description-no-valid-content
   The description element does not contain any valid content (paragraphs, enumerations, etc.).

Signed-off-by: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>
Part-of: <https://gitlab.gnome.org/GNOME/calls/-/merge_requests/781>
---
 data/org.gnome.Calls.metainfo.xml.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/data/org.gnome.Calls.metainfo.xml.in b/data/org.gnome.Calls.metainfo.xml.in
index 6f662abb..c0460eb6 100644
--- a/data/org.gnome.Calls.metainfo.xml.in
+++ b/data/org.gnome.Calls.metainfo.xml.in
@@ -247,16 +247,16 @@
         </description>
       </release>
       <release version="41.0" date="2021-09-18">
-        <description>SIP plugin bugfixes.</description>
+        <description><p>SIP plugin bugfixes.</p></description>
       </release>
       <release version="41~rc" date="2021-09-02">
-        <description>SIP Plugin polish and bugfixes.</description>
+        <description><p>SIP Plugin polish and bugfixes.</p></description>
       </release>
       <release version="41~beta" date="2021-08-14">
-        <description>SIP Plugin usable from the UI.</description>
+        <description><p>SIP Plugin usable from the UI.</p></description>
       </release>
       <release version="41~alpha" date="2021-07-16">
-        <description>First GNOME release</description>
+        <description><p>First GNOME release</p></description>
       </release>
     </releases>
 
-- 
GitLab

